error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/FluxFlatMapUsageTest.java

```diff
@@ -73,7 +67,7 @@ void identification() {
 
   @Test
   void replacementFirstSuggestedFix() {
-    refactoringTestHelper
+    newInstance(FluxFlatMapUsage.class, getClass())
```

Comment:
We can apply the same for the method below as well.
```suggestion
    BugCheckerRefactoringTestHelper.newInstance(FluxFlatMapUsage.class, getClass())
```